webapp/components/CommentForm/CommentForm.vue

```diff
@@ -2,18 +2,18 @@
   <ds-form v-model="form" @submit="handleSubmit">
     <template slot-scope="{ errors }">
       <ds-card>
-        <hc-editor
-          ref="editor"
-          :users="users"
-          :hashtags="null"
```

Comment:
nice, you tested and this is not needed after all, right?

Reply:
Yeah !!!